### PR TITLE
[Shop][Checkout] Displaying detailed shipping fee

### PR DESCRIPTION
**Pull request by tuka217**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

File: features/checkout/shipping_order/seeing_shipping_fee.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @todo
+    Scenario: Seeing the shipping total on order summary
+        Given the store has "UPS" shipping method with "€40.00" fee on fist item and "€5.00" on next 10
+        And I have product "The Sorting Hat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I proceed order with "UPS" shipping method and "Offline" payment
+        Then I should be on the checkout shipping step
+        And I should see shipping fee "€40.00" on first item
+        And I should see shipping fee "€5.00" on next 10 additional items
+
```
Comment (pamil) [inline features/checkout/shipping_order/seeing_shipping_fee.feature]: Unnecessary blank line :>

File: features/checkout/shipping_order/seeing_shipping_fee.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        And I should see shipping fee "€20.00"
+
+    @todo
+    Scenario: Seeing the shipping total on order summary
+        Given the store has "UPS" shipping method with "€5.00" fee per item
+        And I have product "The Sorting Hat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I proceed order with "UPS" shipping method and "Offline" payment
+        Then I should be on the checkout shipping step
+        And I should see shipping fee "€5.00" for every item
```
Comment (pamil) [inline features/checkout/shipping_order/seeing_shipping_fee.feature]: There's only one item in the cart right now.

File: features/checkout/shipping_order/seeing_shipping_fee.feature [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    @todo
+    Scenario: Seeing the shipping total on order summary
+        Given the store has "UPS" shipping method with "€40.00" fee on fist item and "€5.00" on next 10
+        And I have product "The Sorting Hat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I proceed order with "UPS" shipping method and "Offline" payment
+        Then I should be on the checkout shipping step
+        And I should see shipping fee "€40.00" on first item
+        And I should see shipping fee "€5.00" on next 10 additional items
```
Comment (pamil) [inline features/checkout/shipping_order/seeing_shipping_fee.feature]: There's only one item in the cart right now.

**Comment by pjedrzejewski** [comment]
Thank you Ania! 👍

**Pull request merged**